### ISSUE TITLE
Can't compile under 4.03+flambda

### ISSUE THREAD

**Issue by bluddy** [ocaml-community/zed#3]
It looks like an ocamlbuild problem.

**Comment by diml** [comment]
What error do you get? I just installed it on 4.03.0+flambda without problem

**Comment by bluddy** [comment]
```
 File "src/zed_utf8.ml", line 1:
 Error: /Users/yotambarnoy/.opam/4.03.0+flambda/lib/ocaml/uChar.cmx
 contains the description for unit
 UChar when Uchar was expected
```

**Comment by bluddy** [comment]
I didn't see this specific error before you asked me for it, I only saw the warnings on using String.upper and thought it was treating warnings like errors.

**Comment by diml** [comment]
It seems that with flambda all cmx are required. camomile doesn't install them at the moment which seems to be the source of the problem. I did a patch for it and it fixes the problem (ocaml/opam-repository#6400)

**Comment by bluddy** [comment]
Yeah not relevant anymore.
